openapi.yaml

```diff
@@ -89,6 +89,16 @@ paths:
           description: latitude, longitude in degrees
           schema:
             type: string
+        - name: filterType
```

Comment:
(not a maintainer)

I think for both of these usages, something like `type` would be more in-line with the other API-parameters